src/controllers/reviewController.js

```diff
@@ -0,0 +1,43 @@
+const reviewService = require("../services/reviewService");
+
+const addReview = async (req, res) => {
+  const userId = req.params.id
+  const { bookId, review, rating } = req.body;
+  const addReview = await reviewService.addReview(userId, bookId, review, rating)
+  return res.status(201).json({message : addReview});
+} 
+
+const getReview = async (req, res) => {
+  const bookId = req.params.bookId
+  return res.status(200).json(await reviewService.getReview(bookId));
+};
+
+const updateReview = async (req, res) => {
+  const id = req.params.id
+  const { review, rating, userId, bookId } = req.body; 
+    const books = await reviewService.updateReview(
+      id,
+      review, 
+      rating, 
+      userId,
+      bookId
+    ); 
+    return res.status(200).json({ books, message: "Updated" });
+};
+
+const deleteReview = async (req, res) => {
+  const id = req.body.id
+  // const userId = req.params.id
+  const userId = 1
```

Comment:
테스트를 위해 주석 처리를 한 부분이 있는데 확인하고 수정해주세요!

userId는 미들웨어로 적용한 auth 모듈에서 받아올 수 있도록 수정해주면 좋을 것 같습니다.

```suggestion
  const userId = req.user.id;
```

Reply:
수정완료하였습니다!